docs/Developers/Writing_Good_Commits.rst

```diff
@@ -71,7 +71,9 @@ Example:
 
     setup.py: Change bears' entrypoint
 
--  Maximum of 50 characters.
+-  Maximum of 50 characters. 50 characters is not a hard limit, just a rule
```

Comment:
"50 characters is not a hard limit" is a bit misleading.
It *is* a hard limit in coala.
The bot will block any PR which has commits with 51 characters.
This needs to be rephased a bit before it can be merged.